src/components/opt-in-out/helpers/get-learn-redirect-path.ts

```diff
@@ -0,0 +1,18 @@
+import { ProductOption } from 'lib/learn-client/types'
+import { PLATFORM_OPTIONS } from '..'
+
+export function getLearnRedirectPath(
+  currentPath: string,
+  product?: ProductOption
+) {
+  const originalUrl = new URL(currentPath, PLATFORM_OPTIONS['learn'].base_url)
+  const finalUrl = new URL(PLATFORM_OPTIONS['learn'].base_url)
```

Comment:
Redirecting back to learn home page, the middleware on learn will handle the final redirect based on the `path`